src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/data/UserSummaryWithEmail.kt

```diff
@@ -0,0 +1,21 @@
+package uk.gov.justice.digital.hmpps.nomisuserrolesapi.data
+
+import io.swagger.v3.oas.annotations.media.Schema
+
+@Schema(description = "Summary User Information with Email Address")
+data class UserSummaryWithEmail(
```

Comment:
I think perhaps a better approach would be to just have a single UserSummary class and add email to main the `/users` endpoint.

In fact I think we were going to do this at some point any way so "find users" doesn't rely on Auth.

Reply:
agreed, will pick up in next PR